bungiesearch/managers.py

```diff
@@ -45,4 +47,7 @@ def __getattr__(self, alias):
         this is only called as a last resort in case the attribute is not found.
         This function will check whether the given model is allowed to use the proposed alias and will raise an attribute error if not.
         '''
+        if alias[0] == '_':
```

Comment:
This prevents an infinite recursion bug, but I understand why it only happens on python 3+.

1. When initializing, Django copies an "uninitialized" BungiesearchManager object.
2. copy calls [`hasattr(y, '__setstate__')`](https://github.com/python/cpython/blob/f2226aef5cab4d278ff0139670c059e9622a612d/Lib/copy.py#L301), which calls this function.
3. this function calls self.search (via property)
4. self.search calls this function (via `self.model` lookup)
5. infinite recursion.